### ISSUE TITLE
Electon-Updater Error: Unexpected close tag

### ISSUE THREAD

**Issue by nikolaioak** [electron-userland/electron-builder#5344]
Electron-Builder Version: V22.6.0
Electron Version: V9.2.0
Electron-Updater Version: V4.3.5
Building for Windows, NSIS

I've tried looking through all the existing issues and didn't see anything specifically for this. I am testing out a build-release-update cycle and my application is erroring after/when checking for updates. Any help in troubleshooting would be excellent, I'm unsure of the reason it's providing an unexpected close tag error.

I'm using a public repo on GitHub Enterprise and have successfully used electron-builder to build and release the app to GitHub Releases on my enterprise server.

main.log:
`[2020-10-13 10:59:11.688] [info] Checking for update
[2020-10-13 10:59:12.137] [error] Error: Error: Unexpected close tag
Line: 24
Column: 9
Char: >
    at error (C:\Users\nd100573\AppData\Local\Programs\umbsapauto\resources\app.asar\node_modules\sax\lib\sax.js:651:10)
    at strictFail (C:\Users\nd100573\AppData\Local\Programs\umbsapauto\resources\app.asar\node_modules\sax\lib\sax.js:677:7)
    at closeTag (C:\Users\nd100573\AppData\Local\Programs\umbsapauto\resources\app.asar\node_modules\sax\lib\sax.js:871:9)
    at SAXParser.write (C:\Users\nd100573\AppData\Local\Programs\umbsapauto\resources\app.asar\node_modules\sax\lib\sax.js:1436:13)
    at parseXml (C:\Users\nd100573\AppData\Local\Programs\umbsapauto\resources\app.asar\node_modules\electron-updater\node_modules\builder-util-runtime\out\xml.js:161:10)
    at GitHubProvider.getLatestVersion (C:\Users\nd100573\AppData\Local\Programs\umbsapauto\resources\app.asar\node_modules\electron-updater\out\providers\GitHubProvider.js:100:53)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
    at async NsisUpdater.getUpdateInfoAndProvider (C:\Users\nd100573\AppData\Local\Programs\umbsapauto\resources\app.asar\node_modules\electron-updater\out\AppUpdater.js:500:13)
    at async NsisUpdater.doCheckForUpdates (C:\Users\nd100573\AppData\Local\Programs\umbsapauto\resources\app.asar\node_modules\electron-updater\out\AppUpdater.js:516:20)`

My updater.js code:
```
// Modules
const { dialog } = require('electron')
const { autoUpdater } = require('electron-updater')
const isDev = require('electron-is-dev')

// Configure log debugging
autoUpdater.logger = require('electron-log')
autoUpdater.logger.transports.file.level = "debug"

// disable auto downloading of updates
autoUpdater.autoDownload = false

// single export to check for and apply updates
module.exports = () => {
    
    if (isDev) {
        // do nothing
    } else {
        // check for updates from GitHub Releases
        autoUpdater.checkForUpdates()

        // listen for update found
        autoUpdater.on('update-available', () => {
            // prompt user to start download
            dialog.showMessageBox({
                type: 'info',
                title: 'Update Available',
                message: 'A new version of umbsapauto is available. Do you want to update now?',
                buttons: ['Update','No']
            }).then(result => {
                let buttonIndex = result.response
                // if update selected, start download
                if (buttonIndex === 0) {
                    autoUpdater.downloadUpdate()
                }
            })
        })

        // listen for download ready
        autoUpdater.on('update-downloaded', () => {
            dialog.showMessageBox({
                type: 'info',
                title: 'Update Ready',
                message: 'Install and restart now?',
                buttons: ['Yes','Later']
            }).then( result => {
                let buttonIndex = result.response
                // install and restart if yes
                if (buttonIndex === 0) {
                    autoUpdater.quitAndInstall(true,true)
                }
            })
        })
    }
}
```

**Comment by nikolaioak** [comment]
Please let me know if there is a chance you can look at this. Happy to contribute for support.

**Comment by sebek1911** [comment]
@nikolaioak Did you manage to solve this problem? I'm facing the same issue...

**Comment by nikolaioak** [comment]
@sebek1911 no, I ended up transitioning to a generic update server since I already have a few web servers at my disposal, it turned out to be the only option that worked for me. I did try debugging the URL's within the githubprovider code objects to try a handful of things but none of them were successful for me and obviously not supportable long term. The generic update doesn't allow you to publish new versions directly, but I don't mind manually pushing releases to my server.
